src/Sylius/Bundle/ProductBundle/Form/Type/ProductType.php

```diff
@@ -34,6 +34,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('attributes', 'collection', array(
                 'required'     => false,
                 'type'         => 'sylius_product_attribute_value',
+                'prototype' => false,
```

Comment:
I did not want to introduce bigger diff than needed, but yeah, I can adjust that.